src/cppcache/integration-test/CacheHelper.cpp

```diff
@@ -1382,6 +1377,31 @@ void CacheHelper::createDuplicateXMLFile(std::string& originalFile,
          CacheHelper::staticConfigFileList.size());
 }
 
+void CacheHelper::replacePortsInFile(int hostPort1, int hostPort2,
+                                     int hostPort3, int hostPort4, int locPort1,
+                                     int locPort2, const std::string& inFile,
+                                     const std::string& outFile) {
+  std::ifstream in(inFile, std::ios::in | std::ios::binary);
+  if (in) {
+    std::string contents;
+    in.seekg(0, std::ios::end);
+    contents.resize(in.tellg());
```

Comment:
I think you want `reserve()`. `resize()` causes the string to be resized and initialized with `CharT()`. `reserve()` just allocates enough capacity but does not resize or initialize the string.